source/ui/dataman/AMSpectrumAndPeriodicTableView.h

```diff
@@ -156,6 +156,8 @@ protected slots:
 
 	/// The export button.
 	QPushButton *exportButton_;
+	///
```

Comment:
@dretrex Probably because I saw `exportButton_` and got excited, oops!

The connection to the virtual functions occurs in `buildEnergyRangeSpinBoxView()` which is called outside the constructor and after initialization in `AMXRFDetailedDetectorView`, though it is called in the constructor of `AMScanViewUtilities`. It might be worth refactoring the `AMScanViewUtilities` constructor to be more in line with AMXRF?